src/metabase/lib/field.cljc

```diff
@@ -696,13 +694,12 @@
   ([query        :- ::lib.schema/query
     stage-number :- :int
     field-ref    :- some?]
-   (let [stage   (lib.util/query-stage query stage-number)
-         ;; not 100% sure why, but [[lib.metadata.calculation/visible-columns]] doesn't seem to return aggregations,
+   (let [;; not 100% sure why, but [[lib.metadata.calculation/visible-columns]] doesn't seem to return aggregations,
          ;; so we have to use [[lib.metadata.calculation/returned-columns]] instead.
```

Comment:
Aggregations don't fit the understanding of "visible" that we used early on in lib. They're only visible in `orderable-columns` since that post-aggregation.

(This is part of why I think generic `visible-columns` is ill-defined, and should be subsumed by the more context-specific `filterable-columns`, `orderable-columns`, etc.)